pex/pex_bootstrapper.py

```diff
@@ -18,75 +18,86 @@
 __all__ = ('bootstrap_pex',)
 
 
-def find_in_path(target_interpreter):
-  if os.path.exists(target_interpreter):
-    return target_interpreter
-
-  for directory in os.getenv('PATH', '').split(os.pathsep):
-    try_path = os.path.join(directory, target_interpreter)
+def _find_pex_python(pex_python):
+  def try_create(try_path):
     if os.path.exists(try_path):
-      return try_path
+      try:
+        return PythonInterpreter.from_binary(try_path)
+      except Executor.ExecutionError:
+        pass
```

Comment:
More Pythonic to return `None`, which ends up happening implicitly right now. By doing so, the return type of `try_create` becomes very clearly `Optional[PythonInterpreter]`.

Reply:
Done.